src/pages/[platform]/build-a-backend/auth/auth-migration-guide/index.mdx

```diff
@@ -1471,18 +1353,23 @@ NOTE: signIn no longer accepts validationData in v6
     }
     ```
 
-  </div>
-</Columns>
+  </Block>
+  <Block name="V5">
+    ```js
+    import { Auth } from 'aws-amplify';
 
-## Auth.verifyTotpToken
+    const handleSetupTOTP = async ({ user }) => {
+      const secretCode = await Auth.setupTOTP(user);
+      const setupUri = `otpauth://totp/AWSCognito:${user}?secret=${secretCode}&issuer=sample-app`;
+    }
+    ```
+
+  </Block>
+</BlockSwitcher>
 
-```diff
--  import { Auth } from 'aws-amplify';
-+  import { verifyTOTPSetup } from 'aws-amplify/auth';
+## Auth.verifyTotpToken
 
--  Auth.verifyTotpToken(user: CognitoUser | any, challengeAnswer: string) => Promise<CognitoUserSession>;
-+  verifyTOTPSetup: (input: VerifyTOTPSetupInput) => Promise<void>;
-```
+In v6, `Auth.verifyTotpToken` has become `verifyTOTPSetup`, and the input has changed: you no longer need to send in a CognitoUser and the `challengeAnswer` parameter has become the `code` property in the input object. The method also no longer returns a CognitoSession: see [Auth.currentSession](/[platform]/build-a-backend/auth/auth-migration-guide/#authcurrentsession) for details on how you would retrieve the current session details returned in v5.
```

Comment:
Do links with `[platform]` in them work as expected?

Reply:
Yes, they seem to.

Reply:
The spell-check doesn't like them a lot of the time though.